src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts

```diff
@@ -28,12 +30,13 @@ export interface IPropertyMapping {
 export interface IRelevantEventsConfig {
     eventType: string;
     propertyMappings: IPropertyMapping[];
-    result: string; //resulting property we want to display for events (ex. Repair Jobs action)
+    action? : string; //mainly being used for self referential purposes
 }
 
 export interface IConcurrentEventsConfig {
     eventType: string; // the event type we are investigating
-    relevantEventsType: IRelevantEventsConfig[]; // possible causes we are considering  
+    relevantEventsType: IRelevantEventsConfig[]; // possible causes we are considering
+    result: string; //resulting property we want to display for events (ex. Repair Jobs action)
```

Comment:
this should be optional?

Reply:
I thought we were only making the relevanteventtypes result optional?